docs: add Container Registries usage documentation (#1712)
mia-platform/documentation: docs/console/company-configuration/project-blueprint.md

---

Through the "Project Blueprint" section at the Company level, it is possible to set some common properties for all Projects that will be created in that Company.

> The settings are grouped by subject area through dedicated tabs, explained in detail below.

Reviewer: ```suggestion
The settings are grouped by subject area through dedicated tabs, as explained below.
```

## Container Registries

The container registries section allows you to manage the container registries you can use to store the Docker images of [Templates](/marketplace/templates/mia_templates.md) and [Examples](/marketplace/examples/mia_examples.md) created from the Marketplace.

![container registries](./img/container-registries-table.png)

This section allows you to view, add, edit, and delete the container registries that will be added to the Projects of the Company upon creation.

You can add a new container registry by clicking on the "Add Container Registry" button. This will open a modal where you can insert the required information to add a new container registry:

- **Name**: the name of the container registry. It is used to identify the container registry in the Console, you can choose any name you prefer.
- **Registry Hostame**: the hostname of the container registry. It refers to the container registry URL, for example, `docker.io` for Docker Hub.

> - **Image Pull Secret Name**: the name of the Image Pull Secret that will be used by Kubernetes to pull images from the container registry. Notice that the Secret must be present in the Kubernetes cluster where the Project is deployed, the Console won't create it for you.

Reviewer: Could you explain the default switch in this list? 
Why when created first container is set to true? 
If I change this field what happens to the first one?

Reviewer: Idem to project section. 
Could you specify that this container registry will be attached to the next project that the user create in the future?

Author: I explained the default switch.

> Could you specify that this container registry will be attached to the next project that the user create in the future?

I already did that, there is an info box a few lines below https://github.com/mia-platform/documentation/pull/1712/files#diff-c4c9371246141c84271e027f07a2879d8dc9b5e4e6a522c3d9f3db34720cc148R298
